examples/src/main/kotlin/io/grpc/examples/helloworld/HelloWorldClient.kt

```diff
@@ -35,12 +35,8 @@ class HelloWorldClient constructor(
 
   fun greet(name: String) = runBlocking {
     val request = HelloRequest.newBuilder().setName(name).build()
-    try {
-      val response = stub.sayHello(request)
-      println("Greeter client received: ${response.message}")
-    } catch (e: StatusException) {
-      println("RPC failed: ${e.status}")
-    }
+    val response = async { stub.sayHello(request) }
+    println("Received: ${response.await().message}")
```

Comment:
I like this simpler code, but most of the other examples, including the Hello World for Java, uses a try-catch statement. @lowasser, what do you prefer?

Reply:
I personally find it odd to recommend wrapping ever RPC call in a try/catch, but I've seen this pattern in submitted client library samples as well, so I could go either way.

Reply:
Oh, actually just discovered that `sayHello` is a suspend fun so we can just write this as:
```
val request = HelloRequest.newBuilder().setName(name).build()
val response = stub.sayHello(request)
println("Received: ${response.message}")
```
since it is in a `runBlocking` block.  Adding the `async` and `await()` is redundant.

Reply:
Full example that we can update the blog and this code to:
```
  suspend fun greet(name: String) = coroutineScope {
    val request = HelloRequest.newBuilder().setName(name).build()
    val response = stub.sayHello(request)
    println("Received: ${response.message}")
  }
```

WDYT?

Reply:
I'm personally in favor of avoiding the try-catch, since we shouldn't be promoting that folks catch and just print "exception" and let the code carry on. My 2 cents.

Reply:
(But I think that we need to keep the `runBlocking` to avoid pushing out the need for a coroutine scope at the point of call at Line 66 below).

Reply:
Updated to what I think is the most idiomatic use of coroutines, but I'm not certain.

Reply:
I'm happy to skip the try/catch.  My feeling on the async/await is that while it is indeed technically redundant, it makes the point more explicitly that it can be launched asynchronously and that you could, if you wanted, do more work in between these lines while the RPC is running.

Reply:
Cool. Added back the async await.